man/man1/git-hack.1

```diff
@@ -1,7 +1,7 @@
 .TH "GIT-HACK" "1" "04/02/2015" "Git Town 0\&.6\&.0" "Git Town Manual"
 
 .SH "NAME"
-git hack \- cut a new feature branch off the main branch
+git hack \- create a new feature branch
 
 .SH "SYNOPSIS"
 \fIgit hack\fR <branchname> [parent branch name]
```

Comment:
unrelated but I think this should be updated from `[parent branch name]` to `[<parent_branch_name>]`